setup.py

```diff
@@ -77,7 +77,7 @@
         "jupyter_server>=1.7.0",
         "jupyterlab>=3.4.6",  # comment out to use local jupyterlab
         "jupyterlab-lsp>=3.8.0",  # comment out to use local jupyterlab
-        "jupyterlab-git~=0.32",  # Avoid breaking 1.x changes
+        "jupyterlab-git~=0.32",  # Avoid baking 1.x changes
```

Comment:
This seems like a typo.

Reply:
😄  That's very weird... I definitely didn't consciously change this but I'm not sure why my IDE would have done so for me either

Reply:
🍪

Reply:
> I definitely didn't consciously change this but I'm not sure why my IDE would have done so for me either

I've found that pycharm will "touch" unexpected files during refactoring and depending on the selected options - which I find infuriating.  Perhaps you did a "refactor->rename" that dealt with "re"?  No worries though - just mentioning.

Reply:
I bet it was something like that!